pom.xml

```diff
@@ -53,6 +53,11 @@
 			<artifactId>jna</artifactId>
 			<version>5.13.0</version>
 		</dependency>
+		<dependency>
+			<groupId>com.google.code.gson</groupId>
+			<artifactId>gson</artifactId>
+			<version>2.10.1</version>
+		</dependency>
```

Comment:
Added this dependency to resolve import errors and using gson. This reduces technical debt.

https://martinfowler.com/bliki/TechnicalDebt.html
https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3